apps/storybook/stories/Namekit/DisplayedPrice.stories.tsx

```diff
@@ -0,0 +1,365 @@
+import { Currency, PriceCurrencyFormat } from "@namehash/ens-utils";
+import {
+  AltPriceDisplayFormat,
+  DisplayedPrice,
+  PriceDisplayPosition,
+  PriceDisplaySize,
+  CurrencySymbolPosition,
+  CurrencySymbology,
+} from "@namehash/namekit-react";
+import type { Meta, StoryObj } from "@storybook/react";
+
+const meta: Meta<typeof DisplayedPrice> = {
+  component: DisplayedPrice,
+  title: "Namekit/DisplayedPrice",
+  argTypes: {
+    symbolFillColor: { control: "color" },
+    price: {
+      control: "object",
+    },
+    size: {
+      options: PriceDisplaySize,
+      control: { type: "select" },
+    },
+    altPriceDisplayFormat: {
+      options: AltPriceDisplayFormat,
+      control: { type: "select" },
+    },
+    symbolPosition: {
+      options: CurrencySymbolPosition,
+      control: { type: "select" },
+    },
+    currencySymbology: {
+      options: CurrencySymbology,
+      control: { type: "select" },
+    },
+    altPriceDisplaySize: {
+      options: PriceDisplaySize,
+      control: { type: "select" },
+    },
+    altPriceDisplayPosition: {
+      options: PriceDisplayPosition,
+      control: { type: "select" },
+    },
+    priceTextDisplaySize: {
+      options: PriceDisplaySize,
+      control: { type: "select" },
+    },
+  },
+};
+
+declare global {
+  interface BigInt {
+    toJSON(): string;
+  }
+}
+
+BigInt.prototype.toJSON = function () {
+  return `${this.toString()}n`;
+};
+
+export default meta;
+
+type Story = StoryObj<typeof DisplayedPrice>;
+
+export const EthPrice: Story = {
+  args: {
+    price: {
+      currency: Currency.Eth,
+      value: 1000000000000000000n,
+    },
+    altPrice: {
+      currency: Currency.Usd,
+      value: 198900n,
+    },
+    altPriceDisplayFormat: AltPriceDisplayFormat.Tooltip,
+  },
+};
+export const EthPriceWithoutCurrencyTooltip: Story = {
+  args: {
+    price: {
+      currency: Currency.Eth,
+      value: 1000000000000000000n,
+    },
+    altPrice: {
+      currency: Currency.Usd,
+      value: 198900n,
+    },
+    altPriceDisplayFormat: AltPriceDisplayFormat.Tooltip,
+    describeCurrencyInTooltip: false,
+  },
+};
+export const UsdPrice: Story = {
+  args: {
+    price: {
+      currency: Currency.Usd,
+      value: 198900n,
+    },
+    altPrice: {
+      currency: Currency.Eth,
+      value: 1000000000000000000n,
+    },
+    altPriceDisplayFormat: AltPriceDisplayFormat.Tooltip,
+  },
+};
+export const AlternativePriceDisplayedAsTextNextToPrice: Story = {
+  args: {
+    price: {
+      currency: Currency.Usd,
+      value: 198900n,
+    },
+    altPrice: {
+      currency: Currency.Eth,
+      value: 1000000000000000000n,
+    },
+    priceTextDisplaySize: PriceDisplaySize.Large,
+    altPriceDisplayPosition: PriceDisplayPosition.Right,
+    altPriceDisplayFormat: AltPriceDisplayFormat.Text,
+  },
+};
+export const AlternativePriceDisplayedAsTextInBottom: Story = {
```

Comment:
Why does this change the horizontal alignment of the UI component?

Reply:
I centered everything now